library/src/main/java/com/novoda/downloadmanager/lib/DownloadInfo.java

```diff
@@ -193,53 +80,46 @@ private Long getLong(String column) {
      */
     public static final String EXTRA_IS_WIFI_REQUIRED = "isWifiRequired";
 
-    public long mId;
-    public String mUri;
-    /**
-     * Add to check if scannable i.e. we don't want internal files to be scanned
-     */
-    public boolean mScannable;
-    public boolean mNoIntegrity;
-    public String mHint;
-    public String mFileName;
-    public String mMimeType;
-    public int mDestination;
-    public int mVisibility;
```

Comment:
Is this needed for the TODO in https://github.com/novoda/download-manager/blob/8eac8276696c4810e1a81cf6c06d9320c0b15f57/library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java#L674